features/raadpleeg-verblijfplaats-met-periode/gba/test/foutsituaties-gba.feature

```diff
@@ -0,0 +1,382 @@
+      | datumVan            | <datum van>         |
+      | datumTot            | <datum tot>         |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                      |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1 |
+      | title    | Een of meerdere parameters zijn niet correct.               |
+      | status   | 400                                                         |
+      | detail   | De foutieve parameter(s) zijn: datumTot.                    |
+      | code     | paramsValidation                                            |
+      | instance | /haalcentraal/api/brphistorie/verblijfplaatshistorie        |
+      En heeft het object de volgende 'invalidParams' gegevens
+      | code         | name     | reason                            |
+      | invalidParam | datumTot | datumTot moet na datumVan liggen. |
+
+      Voorbeelden:
+      | datum van  | datum tot  |
+      | 2020-01-01 | 2019-12-31 |
+      | 2020-10-14 | 2020-10-14 |
+      | 2020-03-01 | 2020-02-29 |
+
+
+  Rule: parameter exclusiefVerblijfplaatsBuitenland is optioneel en moet een boolean (true of false) als waarde hebben
+    
+    @fout-case
+    Abstract Scenario: De exclusiefVerblijfplaatsBuitenland parameter is geen boolean
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                              | waarde              |
+      | type                              | RaadpleegMetPeriode |
+      | burgerservicenummer               | 000000012           |
+      | datumVan                          | 2020-01-01          |
+      | datumTot                          | 2021-01-01          |
+      | exclusiefVerblijfplaatsBuitenland | <ongeldige waarde>  |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                            |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.1       |
+      | title    | Een of meerdere parameters zijn niet correct.                     |
+      | status   | 400                                                               |
+      | detail   | De foutieve parameter(s) zijn: exclusiefVerblijfplaatsBuitenland. |
+      | code     | paramsValidation                                                  |
+      | instance | /haalcentraal/api/brphistorie/verblijfplaatshistorie              |
+      En heeft het object de volgende 'invalidParams' gegevens
+      | code    | name                              | reason                  |
+      | boolean | exclusiefVerblijfplaatsBuitenland | Waarde is geen boolean. |
+
+      Voorbeelden:
+      | ongeldige waarde |
+      | 0                |
+      | 1                |
+      | waar             |
+      | ja               |
+      | null             |
+      | undefined        |
```

Comment:
wat voor waarde moet hier worden ingevuld? undefined string? dan is dit hetzelfde als de andere ongeldige waarde. undefined waarde? In dit geval is dat hetzelfde als het niet opgeven van de exclusiefVerblijfplaatsBuitenland parameter en dat zou niet moeten resulteren in een 400 fout

Reply:
ik verwijder beide voorbeelden